packages/orbit-components/src/hooks/useInterval/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { useEffect, useLayoutEffect, useRef } from "react";
+
+function useInterval(callback: () => void, delay: number | null) {
+  const savedCallback = useRef(callback);
+
+  useLayoutEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  useEffect(() => {
+    if (!delay && delay !== 0) return undefined;
```

Comment:
0 is a valid value for delay

Reply:
Which `delay` value would satisfy the condition of `!delay && delay !== 0`?

Reply:
Oh, I see now that `delay` dan also be `null`. In that case I recommend the condition `delay === null` instead.

Reply:
sorry, but you probably did not get it. It should return `undefined` for values like `undefined`, `null`, `""`. With your check, it will return `undefined` only when the value is `null`.

So answering your top question about 
> Which delay value would satisfy the condition of !delay && delay !== 0?

it would satisfy all falsy values except `0`.

Reply:
Got it! 👍